pmoves/scripts/split_compose.py

```
@@ -0,0 +1,250 @@
+        print(f"WARNING: Service count mismatch! {len(services) - total} services missing.")
+    else:
+        print("All services accounted for.")
+
+    # ── Validate YAML syntax ──
+    print("\nValidating YAML syntax...")
+    for fpath in sorted(compose_dir.glob("docker-compose.*.yml")):
+        try:
+            with open(fpath) as f:
+                yaml.load(f)
+            print(f"  ✓ {fpath.name}")
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)